pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -815,7 +835,15 @@ func GetOpenshiftBootstrapClusterRoleBindings() []rbacv1.ClusterRoleBinding {
 		rbacv1helpers.NewClusterBinding(ScopeImpersonationRoleName).
 			Groups(AuthenticatedGroup, UnauthenticatedGroup).
 			BindingOrDie(),
+		rbacv1helpers.NewClusterBinding("system:scc:anyuid-user").ClusterRoleBinding,
```

Comment:
don't mutate these in client commands.  These are for the operators to manage.  Create different ones for client commands.

Reply:
There's very little point in keeping them, then. Are you ok with them being created and removed upon issuing the SCC commands?

Reply:
I removed the clusterrolebindings in the latest patchset, they are being created on demand and use the naming from https://github.com/openshift/origin/pull/22781#discussion_r281688286